### DISCUSSION TITLE
Load testing long running websocket connections

### DISCUSSION THREAD

**Discussion by akhiljay** [artilleryio/artillery#1446] [Artillery]
We are looking to load test our servers to see if they can keep 100,000 concurrent WS connections active for more than 12 hours a day. Is this doable with a macbook + artillery CLI setup?

**Comment by hassy** [comment]
No, you will need more than one client

**Reply by akhiljay** [reply]
When you say more than one client? Does it mean a macbook pro may not be capable of opening up more than 10K or something similar concurrent connections active. 

The second question is how does one keep one virtual user's connection active for upto 12 hours? Do we need to put in a think sequence and sent a heartbeat message every 30 seconds to keep the connection alive?

**Comment by akhiljay** [comment]
@hassy do you happen to know what is the max concurrent connections that we can test using a single macbook ?
